android/src/main/java/com/segment/analytics/kotlin/android/AndroidAnalytics.kt

```diff
@@ -62,7 +67,7 @@ public fun Analytics(
 }
 
 // Logger instance that uses the android `Log` class
-object AndroidLogger: Logger("AndroidLogger") {
+object AndroidLogger : Logger("AndroidLogger") {
```

Comment:
Are we adding spaces or was this an accident? I noticed others don't have that space.

Reply:
i ran a auto-lint on the project after the rebase, so all of the files should conform to the same convention